### PR TITLE
feat: add tax amount alongside tax percentage

### PR DESCRIPTION
**Pull request by AdamDowson**
## Description of the changes
Added tax amount (fixed) alongside tax percentage

## Link to Jira

https://app.asana.com/0/1194620076282706/1196258233315537

### CODE REVIEW

**Review by vnnkl — APPROVED** [review]

**Review by yomarion — CHANGES_REQUESTED** [review]
I think we should update the version from 0.0.2 to 0.0.3.

Also, any idea why the diff is so big compared to the changes?

File: packages/data-format/src/format/rnf_invoice/rnf_invoice-0.0.2.json [hunk 1]
```diff
@@ -133,6 +133,9 @@
           "taxPercent": {
             "type": "number"
           },
+          "taxAmount": {
```
Comment (benjlevesque) [inline packages/data-format/src/format/rnf_invoice/rnf_invoice-0.0.2.json:136]: open question: should we leave the taxPercent a required field with this new one?

File: packages/data-format/test/data/example-invalid-0.0.2.json [hunk 2]
```diff
@@ -1,80 +1,82 @@
...
-      "note": "payment before chrismas"
-    }
-  },
-
-  "miscellaneous": {
-    "manufacturerCompany": "Victor Hugo & Co.",
-    "deliveryCompany": "Gavroche Express"
-  }
-}
+{
```
Comment (benjlevesque) [inline packages/data-format/test/data/example-invalid-0.0.2.json:1]: only formatting for these files?

**Review by benjlevesque — COMMENTED** [review]
I feel this should be a new version of the format, what do you think @rittme ?

File: packages/data-format/src/format/rnf_invoice/rnf_invoice-0.0.2.json (hunk 1 above)
Comment (rittme) [reply packages/data-format/src/format/rnf_invoice/rnf_invoice-0.0.2.json:136]: I agree, we have to handle it better. If possible, having both being mutualy exclusive and mandatory could work.
Another option would be having a `taxAmount` field, and a `taxType` field, where we specify if it's percentage or amount.

**Comment by AdamDowson** [comment]
> I feel this should be a new version of the format, what do you think @rittme ?

Yes. I forgot to post a comment, do you guys think this be a new version? Pros and cons to both @rittme @vnnkl @yomarion

File: packages/data-format/test/data/example-meta-unknown.json [hunk 3]
```diff
@@ -1,78 +1,80 @@
...
-      "note": "payment before chrismas"
-    }
-  },
-
-  "miscellaneous": {
-    "manufacturerCompany": "Victor Hugo & Co.",
-    "deliveryCompany": "Gavroche Express"
-  }
-}
+{
```
Comment (rittme) [inline packages/data-format/test/data/example-meta-unknown.json:1]: why are we changing everything on these files?

File: packages/data-format/test/data/example-invalid-0.0.2.json (hunk 2 above)
Comment (AdamDowson) [reply packages/data-format/test/data/example-invalid-0.0.2.json:1]: I'll try and figure out why the diffs are so crazy here. It was a one line change to the tests

File: packages/data-format/test/data/example-meta-unknown.json (hunk 3 above)
Comment (AdamDowson) [reply packages/data-format/test/data/example-meta-unknown.json:1]: It's a formatting issue - taking a look to see why its happened

**Pull request closed without merging**